### ISSUE TITLE
Register checkpoint issue

### ISSUE THREAD

**Issue by mathrb** [determined-ai/determined#3651]
Hello

I'm having trouble registering a checkpoint into the model registry.
When registering the checkpoint from the experiment view, I got a HTTP 400 error:

```json
{"error":{"code":3,"reason":"InvalidArgument","error":"json: cannot unmarshal number into Go value of type string"}}
```

here is the request body

```json
{"checkpointUuid":"0792233c-0f47-4579-9462-7e16eb97f164","comment":"","labels":[],"metadata":{},"modelName":1,"name":"1.0"}
```

and here is the endpoint: `http://192.168.55.150:8080/api/v1/models/1/versions`

Any idea why I get this error?

Kind regards

**Comment by rb-determined-ai** [comment]
Hi @mathrb!  Sorry we missed this issue and didn't respond sooner.  We're usually more responsive in our slack #support channel.  [Here is a link to sign up](https://join.slack.com/t/determined-community/shared_invite/zt-cnj7802v-KcVbaUrIzQOwmkmY7gP0Ew).

The message says it can't unmarshal a number into a string, and since the only number you have is `"modelName":1` I would guess you just need to use `"modelName":"1"` instead.

That said... I would highly recommend avoiding using our REST APIs directly.  Is there a reason you can't just use the [python sdk](https://docs.determined.ai/latest/interact/api-experimental-client.html#determined.experimental.client.Model.register_version) for this?

**Comment by mathrb** [comment]
Hello,
Actually I'm getting this error via the web UI. I saw the error in the network tab of the web developer tools (Firefox)

**Comment by rb-determined-ai** [comment]
Oh, interesting!  I'll share this with our webui team in that case.

**Comment by mapmeld** [comment]
Hi @mathrb ! Thanks for reporting this issue. This was fixed in the most recent release (0.17.9).
If you want to fix this locally without updating, the relevant one-line change is #3528
